resources/views/cart/item.blade.php

```diff
@@ -27,7 +27,7 @@ class="mx-auto"
                     </div>
                     <div v-for="option in config.cart_attributes">
                         <template v-if="item.product.attribute_values?.[option] && typeof item.product.attribute_values[option] === 'object'">
-                            @{{ option }}: <span v-html="item.product.attribute_values[option]?.join(', ')"></span>
+                            @{{ item.product.attribute_values[option].label }}: <span v-html="item.product.attribute_values[option].options?.join(', ')"></span>
```

Comment:
Might placing the label here into a separate variable simplify the code?
We're memoizing the result in fetchAttributeValues, so we could put it into a computedAsync and get the label straight in the frontend here. Instead of from the product